src/app/[locale]/_utils/idps.ts

```diff
@@ -85,4 +85,5 @@ const IDPS: { identityProviders: IdentityProvider[]; richiediSpid: string } = {
 
 export { IDPS };
 
-export const isIDPKnown = true;
+// TODO remove this temporary flag isIDPKnown after getSessionsList API is ready in a future version
+export const isIDPKnown = process.env.NEXT_PUBLIC_FEATURE_FLAG;
```

Comment:
Here we could make use of an utility function to convert 'true'/'false' value string in boolean values. What do you think about that?

Reply:
made the utility function to convert env string value to boolean type value